app/emails/email.py

```diff
@@ -0,0 +1,17 @@
+from flask_mail import Message
+from app import mail
+from flask import render_template
+from flask import current_app
```

Comment:
You can put these two imports into a single line.

Reply:
done